src/server/connectors/registration/registration.connector.js

```diff
@@ -3,13 +3,13 @@ const { SUBMIT_URL, API_SECRET, CLIENT_NAME } = require("../../config");
 const { logEmitter } = require("../../services/logging.service");
 const { registrationDouble } = require("./registration.double");
 
-const sendRequest = async body => {
+const sendRequest = async (submissionData, pathConfig) => {
```

Comment:
small comment but is there any reason to pass through the whole path config and not simply the id in this case? not urgent to fix just curious

Reply:
It was originally because I thought we'd be sending through the whole thing including Notify template keys etc in the future - but as we're not, and it's just the version, I'll change it!